adapter/internal/oasparser/envoyconf/listener.go

```diff
@@ -317,3 +317,18 @@ func getTracing(conf *config.Config) (*hcmv3.HttpConnectionManager_Tracing, erro
 
 	return tracing, nil
 }
+
+func getListenerCodecType(codecType string) hcmv3.HttpConnectionManager_CodecType {
+	switch codecType {
+	case "AUTO":
+		return hcmv3.HttpConnectionManager_AUTO
+	case "HTTP1":
+		return hcmv3.HttpConnectionManager_HTTP1
+	case "HTTP2":
+		return hcmv3.HttpConnectionManager_HTTP2
+	case "HTTP3":
```

Comment:
Is Http3 tested ?

Reply:
no, even though I added it here. Didn't mention it in the config.toml options. Didn't mention it in the docs as well. shall I remove it from here as well?